lang/sbcl/Portfile

```diff
@@ -41,8 +41,12 @@ checksums       rmd160  95a02d7a489137602ebba333b83bfc1d9eb794be \
                 sha256  89c90720cf9d05dbcd90d690e381a2514c0f1807159e0d7222220c5a8c2d5186 \
                 size    7474392
 
-if {${os.major} < 10} {
+if {${os.major} < 10 || ${build_arch} eq "i386"} {
```

Comment:
Even though we're only supporting Darwin for this port, it's still good practice to surround the version check in a `platform darwin {` block.

Reply:
@mascguy indeed.